### PR TITLE
add extrinsic: unregister_all_enclave

### PR DESCRIPTION
**Pull request by BillyWooo**
as title, as one extrinsic to teerex pallet to unregister all enclaves.

### CODE REVIEW

File: pallets/teerex/src/lib.rs [hunk 1]
```diff
@@ -666,6 +667,20 @@ pub mod pallet {
...
+
+		#[pallet::call_index(32)]
+		#[pallet::weight((195_000_000, DispatchClass::Normal, Pays::No))]
+		pub fn unregister_all_enclave(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+			let sender = ensure_signed(origin)?;
+			ensure!(Some(sender) == Self::admin(), Error::<T>::RequireAdmin);
+
+			for index in <EnclaveRegistry<T>>::iter_keys() {
+				<EnclaveRegistry<T>>::remove(index);
+			}
```
Comment (Kailai-Wang) [inline pallets/teerex/src/lib.rs:679]: There's a `clear` fn

File: pallets/teerex/src/lib.rs [hunk 2]
```diff
@@ -666,6 +667,20 @@ pub mod pallet {
...
+
```
Comment (Kailai-Wang) [inline pallets/teerex/src/lib.rs:684]: We need to adjust `EnclaveIndex` and `EnclaveCount` too

**Comment by BillyWooo** [comment]
Put is on hold. Maybe need more structure refactor.

**Comment by Kailai-Wang** [comment]
We can close this one - we have similar interfaces in teebag

**Pull request closed without merging**